components/Feedback.js

```diff
@@ -40,7 +42,29 @@ const Feedback = ({ author, text, createdAt, provider, isLast, settings }) => {
           {format(parseISO(createdAt), 'PPpp')}
         </Text>
       )}
-      <Text color={textColor[colorMode]}>{text}</Text>
+      <Box color={textColor[colorMode]}>
+        <ReactMarkdown
+          source={text}
+          renderers={{
+            paragraph: MDXComponents.p,
```

Comment:
This isn't great yet, buuuut it's better than nothing, so I'm just gonna go for it.